lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1191,9 +1192,21 @@ def set_proj_type(self, proj_type, focal_length=None):
                                  f"None for proj_type = {proj_type}")
             self._focal_length = np.inf
 
-    def _roll_to_vertical(self, arr):
-        """Roll arrays to match the different vertical axis."""
-        return np.roll(arr, self._vertical_axis - 2)
+    def _roll_to_vertical(
+        self, arr: "np.typing.ArrayLike", sign: Literal[1, -1] = 1
+    ) -> np.ndarray:
+        """
+        Roll arrays to match the different vertical axis.
+
+        Parameters
+        ----------
+        arr : ArrayLike
+            Array to roll.
+        sign : Literal[1, -1], default: 1
+            Roll the array elements in a positive or negative
+            direction. Defaults to the positive direction.
```

Comment:
My point was that the naming `sign` itself is not good because it's on a technical level not reflecting the underlying semantics. Expanding the documentation of the technical action only marginally improves understandability. To be explicit: Please change the name.

Reply:
sign was inspired by [np.sign](https://numpy.org/doc/stable/reference/generated/numpy.sign.html) for reference. 
The arguments I see against `reverse: bool` is that it leads to more code complexity (an additional if-check). This function is triggered every time on move so some performance consideration has to be made in my opinion. Probably not the worst bottleneck though.